src/components/CommandBar/CommandBarSelectionInput.tsx

```diff
@@ -139,8 +139,8 @@ function CommandBarSelectionInput({
         data: {
           selectionType: 'singleCodeCursor',
         },
-      })
-    setHasClearedSelection(true)
+      }) &&
+      setHasClearedSelection(true)
```

Comment:
@franknoirot I got it, I think the lack of && here made https://github.com/KittyCAD/modeling-app/blob/7577053189a1f392ed79dab006944b1e52eb58ef/src/components/CommandBar/CommandBarSelectionInput.tsx#L163-L164 fire more than needed and leading to these skips

Reply:
I noticed after adding this log. With this extra &&, I see it firing only when clicking on other args now.

<img width="717" alt="image" src="https://github.com/user-attachments/assets/4a019a4e-361c-4e88-89dc-1e1540b50727" />